Localizations/en_US.json

```diff
@@ -173,39 +173,39 @@
     "shader_asset_format_disassembly": "Disassembly",
     "shader_asset_format_disassembly_description": "Export the shader as disassembly. Experimental and prone to breakage. This does not compile in the editor.",
     "shader_asset_format_dummy": "Dummy Shader",
-    "shader_asset_format_dummy_description": "Export the shader as a dummy shader. Although it preserves data like the Properties and FallBack, it uses general, opaque shader code.",
-    "shader_asset_format_yaml": "Yaml Asset",
-    "shader_asset_format_yaml_description": "Export the shader as a yaml asset. Experimental and only usable for viewing in the editor. The editor can randomly corrupt these files.",
+    "shader_asset_format_dummy_description": "Export the shader as a dummy shader. Although it preserves data like the Properties and Fallback, it uses general, opaque shader code.",
+    "shader_asset_format_yaml": "YAML Asset",
+    "shader_asset_format_yaml_description": "Export the shader as a YAML asset. Experimental and only usable for viewing in the editor. The editor can randomly corrupt these files.",
     "skip_streaming_assets": "Skip StreamingAssets Folder",
     "sprite_export_title": "Sprite Export Format",
     "sprite_format_native": "Unity",
-    "sprite_format_native_description": "Export in the unity sprite format. Cannot be viewed outside of unity.",
+    "sprite_format_native_description": "Export in the Unity sprite format. Cannot be viewed outside of Unity.",
     "sprite_format_texture": "Texture",
-    "sprite_format_texture_description": "Export as an image of the Sprite Sheet. Can be viewed outside of unity, but slower to export.",
-    "sprite_format_yaml": "Yaml",
-    "sprite_format_yaml_description": "Export as yaml assets which can be viewed in the editor. This is the only mode that ensures a precise recovery of all metadata of sprites.",
+    "sprite_format_texture_description": "Export as an image of the sprite sheet. Can be viewed outside of Unity, but slower to export.",
+    "sprite_format_yaml": "YAML",
+    "sprite_format_yaml_description": "Export as YAML assets which can be viewed in the editor. This is the only mode that ensures a precise recovery of all metadata of sprites.",
     "submesh_count": "Submesh Count",
     "success": "Success!",
     "terrain_export_title": "Terrain Export Format",
     "terrain_format_heatmap": "Heightmap",
-    "terrain_format_heatmap_description": "Export a heatmap of the height of the terrain at each location. Only really useful if you don't care about the details or having the terrain in 3D.",
+    "terrain_format_heatmap_description": "Export a heat map of the height of the terrain at each location. Only really useful if you don't care about the details or having the terrain in 3D.",
```

Comment:
Both "heatmap" and "heat map" are correct. We'll continue using "heatmap."

https://en.wikipedia.org/wiki/Heat_map

Reply:
Yes, both forms are correct. Just like the word "spritesheet," it can be written as "spritesheet" or "sprite sheet," and since you had used "sprite sheet" earlier, I just wanted to maintain consistency. But if you prefer, I can leave it as "heatmap."

Reply:
In the case of "sprite sheet," I think "spritesheet" is less correct because both Google and the web browser spell check are suggesting I insert the space.

Reply:
Really? For some reason, Google doesn't correct me when I search for "spritesheet". But anyway, I understand your concern, and I'll leave it the way it was before.